pkg/obfuscate/sql_test.go

```diff
@@ -1982,8 +1982,7 @@ func TestCassQuantizer(t *testing.T) {
 	}
 }
 
-//nolint:revive // TODO(APM) Fix revive linter
-func TestUnicodeDigit(t *testing.T) {
+func TestUnicodeDigit(_ *testing.T) {
```

Comment:
this is a weird test since it's not making any explicit assertions - I guess it's making sure we don't get stuck or panic on that specific character? 🤔  no change needed here but if we're sure that's why this test exists it might be nice to have a comment explaining that's the purpose of the test?